pkg/cmd/server/kubernetes/node_config_test.go

```diff
@@ -6,24 +6,116 @@ import (
+			EventRecordQPS:              5.0,
+			EnableCustomMetrics:         false,
+			EnableDebuggingHandlers:     true,
+			EnableServer:                true,
+			FileCheckFrequency:          unversioned.Duration{20 * time.Second}, // overridden
+			HealthzBindAddress:          "127.0.0.1",                            // disabled
+			HealthzPort:                 10248,                                  // disabled
+			HostNetworkSources:          "*",                                    // overridden
+			HostPIDSources:              "*",                                    // overridden
+			HostIPCSources:              "*",                                    // overridden
+			HTTPCheckFrequency:          unversioned.Duration{20 * time.Second}, // disabled
+			ImageMinimumGCAge:           unversioned.Duration{2 * time.Minute},
+			ImageGCHighThresholdPercent: 90,
+			ImageGCLowThresholdPercent:  80,
+			LowDiskSpaceThresholdMB:     256,
+			MasterServiceNamespace:      "default",
+			MaxContainerCount:           240,
+			MaxPerPodContainerCount:     2,
+			MaxOpenFiles:                1000000,
+			MaxPods:                     110, // overridden
+			MinimumGCAge:                unversioned.Duration{1 * time.Minute},
+			NetworkPluginDir:            "/usr/libexec/kubernetes/kubelet-plugins/net/exec/",
+			NetworkPluginName:           "", // overridden
+			NonMasqueradeCIDR:           "10.0.0.0/8",
+			VolumePluginDir:             "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/",
+			NodeStatusUpdateFrequency:   unversioned.Duration{10 * time.Second},
+			NodeLabels:                  map[string]string{},
+			OOMScoreAdj:                 -999,
+			LockFilePath:                "",
+			PodInfraContainerImage:      kubetypes.PodInfraContainerImage, // overridden
+			Port:                           10250, // overridden
+			ReadOnlyPort:                   10255, // disabled
+			RegisterNode:                   true,
+			RegisterSchedulable:            true,
+			RegistryBurst:                  10,
+			RegistryPullQPS:                5.0,
+			KubeletCgroups:                 "",
+			RktPath:                        "",
+			RktStage1Image:                 "",
+			RootDirectory:                  "/var/lib/kubelet", // overridden
+			RuntimeCgroups:                 "",
+			SerializeImagePulls:            true,
+			StreamingConnectionIdleTimeout: unversioned.Duration{4 * time.Hour},
+			SyncFrequency:                  unversioned.Duration{1 * time.Minute},
+			SystemCgroups:                  "",
+			TLSCertFile:                    "", // overridden to prevent cert generation
+			TLSPrivateKeyFile:              "", // overridden to prevent cert generation
+			ReconcileCIDR:                  true,
+			KubeAPIQPS:                     5.0,
+			KubeAPIBurst:                   10,
+			ExperimentalFlannelOverlay:     false,
+			OutOfDiskTransitionFrequency:   unversioned.Duration{5 * time.Minute},
+			HairpinMode:                    "promiscuous-bridge",
```

Comment:
Hm... it looks like the current default options (`HairpinMode: "promiscuous-bridge", ConfigureCBR0: false`) will result in `pkg/kubelet/kubelet.go:effectiveHairpinMode()` logging a warning and then switching to `"hairpin-veth"` mode, which is the old behavior. So no change in behavior, but we get a warning now. So we should override it to either `"none"` (which is what openshift-sdn wants, since it doesn't use a bridge), or `"hairpin-veth"` (which is maybe what some non-openshift-sdn plugins might expect, and doesn't actually hurt openshift-sdn now that the spurious "could not set hairpin_mode" warning has been fixed).